DigitalLearningSolutions.Web/Styles/home/learningContent.scss

```diff
@@ -0,0 +1,35 @@
+@import "homeShared";
+
+.learning-content-item__header {
+  @include govuk-media-query($until: tablet) {
+    margin-top: 16px;
+    text-align: center;
+  }
+}
+
+.learning-content-item__text {
+  @include govuk-media-query($until: tablet) {
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+  }
+}
+
+.learning-content-item__details-container {

```

Comment:
Just wanted to double check that none of these NHS components do this already?
https://service-manual.nhs.uk/design-system/styles/layout

Reply:
Nothing there seems perfectly suited to making a column fit the width of the image, but I may have missed something...

Reply:
Yeah, looks like they're mainly using grid layout for this, but it might not be suitable for this case: https://service-manual.nhs.uk/design-system/styles/layout